setup.py

```diff
@@ -37,20 +37,20 @@
 """
 
 import glob
-import os.path
+import os
 import subprocess
 import sys
 
 from distutils.spawn import find_executable
 from pkg_resources import get_build_platform
-from setuptools import setup, find_packages
+from setuptools import setup
 from setuptools.extension import Extension
 
 # Execute DAKOTA to get version.
 try:
```

Comment:
I disagree. The try shouldn't wrap a logical "thing", it should be around whatever that *can* throw the exception in question, and preferably nothing more. If that whatever is functions (which both throw the same exception or handful of exceptions).

They also shouldn't bubble unless you *want* that as a part of your interface (which you often do, but sometimes not).